libraries/AP_GPS/GPS_Backend.h

```diff
@@ -61,6 +61,7 @@ class AP_GPS_Backend
     void broadcast_gps_type() const;
     virtual void Write_DataFlash_Log_Startup_messages() const;
 
+    virtual bool is_healthy() const { return true; }
```

Comment:
it's a virtual. When the driver does not implement it then this function is used as a default.